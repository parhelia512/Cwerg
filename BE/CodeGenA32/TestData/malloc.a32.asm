# malloc

# ========================================
.fun main NORMAL [S32] = []
.bbl start
    mov size:U32 1
.bbl loop
    pusharg size
    bsr malloc
    poparg addr:A32
    bitcast x:U32 addr
    pusharg x
    bsr print_x_ln
    mul size size 4
    blt size 0x1000000 loop
    pusharg 0:S32
    ret
