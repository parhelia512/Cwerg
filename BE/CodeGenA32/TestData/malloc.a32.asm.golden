31000
31010
31020
31030
31070
31170
31570
32570
36570
46570
86570
186570
